docs/install.rst

```diff
@@ -10,18 +10,14 @@ Requirements
 Package installation
 --------------------
 
-Python package::
+Install using pip:
 
-    pip install django-auth-adfs
+    pip install django-guid
```

Comment:
Oh wow, says a lot about how much I have thought about or been inspired by that package..